moduleframework/helpers/container_helper.py

```diff
@@ -37,41 +37,50 @@ def __init__(self):
         set basic object variables
         """
         super(ContainerHelper, self).__init__()
-        static_name="testcontainer"
+        static_name = "testcontainer"
         self.tarbased = None
-        self.jmeno = None
+        self.name = None
         self.docker_id = None
-        self.icontainer = self.get_url()
-        if not self.icontainer:
+        self._icontainer = self.get_url()
+        if not self._icontainer:
             raise ConfigExc("No container image specified in the configuration file or environment variable.")
-        if ".tar" in self.icontainer:
-            self.jmeno = static_name
+        if ".tar" in self._icontainer:
+            self.name = static_name
             self.tarbased = True
-        if "docker=" in self.icontainer:
-            self.jmeno = self.icontainer[7:]
+        if "docker=" in self._icontainer:
+            self.name = self._icontainer[7:]
             self.tarbased = False
         else:
             # untrusted source
             self.tarbased = False
-            self.jmeno = self.icontainer
+            self.name = self._icontainer
         self.docker_static_name = ""
         if get_if_reuse():
             self.docker_static_name = "--name %s" % static_name
 
-    def getURL(self):
+    @property
+    def URL(self):
         """
         It returns actual URL link string to container, It is same as URL
 
         :return: str
         """
-        return self.icontainer
+        return self._icontainer
+
+    @URL.setter
+    def URL(self, value):
+        self._icontainer = value
+
+    @URL.getter
```

Comment:
you already define
```python
@property
def URL(...):
```

so you can remove this explicit getter

Reply:
I see, I though that `getter` should be defined always. THX.